<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="imgLogo">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="DBTool.qrc">:/imgLogo/KiwiSun.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabStart">
         <attribute name="title">
          <string>Start</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="lblP1Step1">
              <property name="text">
               <string>1. Select program type :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbProgramKiwiSun">
              <property name="text">
               <string>KiwiSun</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbProgramSensolite">
              <property name="text">
               <string>Sensolite</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="lblP1Step2">
              <property name="text">
               <string>2. Select directory :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ledPathDB">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbExpSelectDir">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="DBTool.qrc">
                <normaloff>:/imgButton/open.png</normaloff>:/imgButton/open.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="lblP1Step3">
              <property name="text">
               <string>2. Import data files :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbImportDB">
              <property name="text">
               <string>Import database</string>
              </property>
              <property name="icon">
               <iconset resource="DBTool.qrc">
                <normaloff>:/imgButton/import.png</normaloff>:/imgButton/import.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabPatientCardTypeImport">
         <attribute name="title">
          <string>Patientcard types</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listLog">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>171</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>171</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbExitApplication">
          <property name="text">
           <string>  Exit application  </string>
          </property>
          <property name="icon">
           <iconset resource="DBTool.qrc">
            <normaloff>:/imgButton/exit.png</normaloff>:/imgButton/exit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DBTool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbExitApplication</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
